src/transformers/models/auto/processing_auto.py

```diff
@@ -41,6 +41,7 @@
 
 PROCESSOR_MAPPING_NAMES = OrderedDict(
     [
+        ("blip", "BLIPProcessor"),
```

Comment:
You should also add defaults in `tokenization_auto`

Reply:
Fixed in [9afcd0e](https://github.com/huggingface/transformers/pull/20716/commits/9afcd0efd9de462c1b5f13574bd031974dbcc4ec)